data/ignore_ids_safety_scan.json

```diff
@@ -411,6 +413,7 @@
             "py2": {
             },
             "py3": {
+                "51159":"in order to to get approved version of cryptography (>=39.0.0), the python version needs to be 3.9 or above."
```

Comment:
That is a good point. course, with 23.9 or above, the version "should" already be updated, but if it didn't we would miss it with this.

Reply:
Note: this is not something we will solving on this PR. but will discuss offline on approaches.

Reply:
@xncqr I noticed here https://pypi.org/project/cryptography/#history there is no version 39.0.0 - is this the right message (related to python 3.9), or is it simply the fact that there is no "approved" version of cryptography package? Can you link the information that led you to add this to the ignore list?

Reply:
please looks up.. it's supposed to be py3.9 and the comment was updated this morning.

Reply:
In terms of the 39.0.0.. that is correct as released to the CVE. in that context 39.0.0 is referenced to py3.9 with subversion. So that is correct.

Reply:
Can you please link the CVE?

Reply:
@arjkesh the skip is added to all the images based on the `python_version` defined in the buildspec file, for eg., https://github.com/aws/deep-learning-containers/blob/master/pytorch/buildspec.yml#L54.

In case we define the `python_version` for Py39 images as py3, it will apply on the Py39 images, otherwise not.

Reply:
@shantanutrip question. can we be more specific in the json file to state the next level version? aka py37 over py3?